src/getHumanAge.test.js

```diff
@@ -1,8 +1,96 @@
+'use strict';
+
 describe('getHumanAge', () => {
-  const { getHumanAge } = require('./getHumanAge');
+  const {
+    getHumanAge,
+  } = require('./getHumanAge');
 
   test('should be declared', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return zeros if cats age and dogs age < 15', () => {
```

Comment:
i had the same stuff in test summary but Tanya rejected it and the reason was to use specific values (not cats age and dogs age < 15 but catAge = 7 and dogAge = 11)

Reply:
Do you mean that specific values are needed right in the summary?

'Should return zeros if cats age and dogs age < 15'  == > 'should return zeros if ca = 7 and da = 11'
?

Reply:
yep